.github/workflows/deploy.yml

```
@@ -41,6 +41,21 @@ jobs:
       run: |
         gcloud auth configure-docker
 
+    - name: Set up Node.js
+      uses: actions/setup-node@v3
```

actionlint (1.7.7)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)